sdk/src/main/java/io/dapr/client/DaprClientImpl.java

```diff
@@ -1295,6 +1305,96 @@ public Mono<UnsubscribeConfigurationResponse> unsubscribeConfiguration(Unsubscri
     }
   }
 
+  @Override
+  public <T> Mono<Void> scheduleJobAlpha1(Job<T> job) {
+    try {
+  	  final String name = job.getName();
+  	  final T data = job.getData();
+  
+  	  if (name == null || name.trim().isEmpty()) {
+  		  throw new IllegalArgumentException("Job name cannot be null or empty");
+  	  }
+  	  if (data == null) {
+  		  throw new IllegalArgumentException("Job data cannot be empty");
+  	  }
+  
+  	  DaprProtos.Job.Builder jobBuilder = DaprProtos.Job.newBuilder()
+  			  .setName(name);
+  	  if (data instanceof String) {
+  		  jobBuilder.setData(Any.newBuilder().setValue(ByteString.copyFrom((String) data, CHARSET)));
+  	  } else if (data instanceof byte[]) {
+  		  String base64 = OBJECT_MAPPER.writeValueAsString(data);
+  		  jobBuilder.setData(Any.newBuilder().setValue(ByteString.copyFrom(base64, CHARSET)));
+  	  } else {
+  		return Mono.error(() -> {
+            throw new IllegalArgumentException("Job data value must be String or byte[]");
+          });
+  	  }
+  	  if (job.getSchedule() != null && !job.getSchedule().trim().isEmpty()) {
+  		  jobBuilder.setSchedule(job.getSchedule());
+  	  }
+  	  if (job.getRepeats() != null) {
+  		  jobBuilder.setRepeats(job.getRepeats());
+  	  }
+  	  if (job.getDueTime() != null && !job.getDueTime().trim().isEmpty()) {
+  		  jobBuilder.setDueTime(job.getDueTime());
+  	  }
+  	  if (job.getTtl() != null && !job.getTtl().trim().isEmpty()) {
+  		  jobBuilder.setTtl(job.getTtl());
+  	  }
+  
+  	  DaprProtos.ScheduleJobRequest.Builder builder =
+  			  DaprProtos.ScheduleJobRequest.newBuilder()
+  			  .setJob(jobBuilder.build());
+  
+  	  return this.<DaprProtos.ScheduleJobResponse>createMono(
+  			  it -> intercept(null, asyncStub).scheduleJobAlpha1(builder.build(), it))
+  			  .then();
+    } catch (Exception ex) {
+  	  return DaprException.wrapMono(ex);
+    }
+  }
+  
+  @SuppressWarnings("unchecked")
+  @Override
+  public <T> Mono<Job<T>> getJobAlpha1(String name, Class<T> clazz) {
+	  try {
+		  if (name == null || name.trim().isEmpty()) {
+	  		  throw new IllegalArgumentException("Job name cannot be null or empty");
+	  	  }
+		  return this.<DaprProtos.GetJobResponse>createMono(
+				  it -> intercept(null, asyncStub).getJobAlpha1(DaprProtos.GetJobRequest.newBuilder().setName(name).build(), it))
+		  .map(it -> {
+			  DaprProtos.Job _job = it.getJob();
+			  T data = null;
+			  if (clazz.isInstance(String.class)) {
+				  data = (T)_job.getData().toByteString().toString(CHARSET);
+			  } else if (clazz.isInstance(byte[].class)) {
+				  data = (T) _job.getData().toByteArray();
+			  } else {
+				  throw new IllegalArgumentException("Job data type must be String or byte[]");
+			  }
+			  return new Job<>(_job.getName(), _job.getSchedule(), _job.getRepeats(), _job.getDueTime(), _job.getTtl(), data);
+		  });
+	  } catch (Exception ex) {
+		  return DaprException.wrapMono(ex);
+	  }
+  }
+  
```

Comment:
@salaboy @artur-ciocanu this getJobAplha1 implementation is fine? I taking in the type of the data.